packages/react/src/components/UIShell/__stories__/UIShell.stories.js

```diff
@@ -1153,7 +638,7 @@ export const DemoPanel = () => {
                     renderIcon={Layers}>
                     Infrastructure
                   </SideNavLink>
-                  <SideNavTitle>Experience services</SideNavTitle>
+                  <SideNavDivider />
```

Comment:
Can we bring back the `SideNavTitle` guessing it got changed from the merge?

Reply:
fixed! sorry about that